.idea/misc.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11.0.3" project-jdk-type="JavaSDK" />
```

Comment:
@anandnagar this explicitly treats our project's minimum SDK as Java 8 .
sourceCompatibility
Defines which language version of Java your source files should be treated as.

targetCompatibility
Defines the minimum JVM version your code should run on, i.e. it determines the version of byte code the compiler generates.

So I think we need this, let me know if you think otherwise